scripts/test_with_ganache.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+node ./node_modules/ganache-cli/build/cli.node.js --gasLimit 650000000 --gasPrice 0 >/dev/null 2>/dev/null &
+OUTPUT="$(node ./node_modules/solium/bin/solium.js -d contracts/)"
+ 
+if [ $? -eq 1 ]; then
+	>&2 echo "Error while linting"
+	>&2 echo $OUTPUT
+    exit 1
+fi
+
+echo $OUTPUT | grep "warning"
```

Comment:
Grepping for `warning` and `error` seems *super* fragile.

Solium *currently* appears to print one or the other: https://github.com/duaraghav8/Solium/blob/6cf5340f226e299bbe92d98c895355d0609f1267/config/rulesets/solium-all.js

...but do they guarantee this will always be the case?

Reply:
```
After linting over your code, Solium produces either warnings, errors or both. The tool exits with a non-zero code
only if 1 or more errors were found. So if all you got was warnings, solium exits with 0.
```

Ugh.

Reply:
Let's add an additional test for a non-zero return value. If it returns non-zero, then exit 1. Perhaps this makes us a little more future-proof.

Gross that they don't appear to have a stable means to identify when something went wrong.